libexec/rtld-elf/rtld_c18n.h

```diff
@@ -122,6 +122,15 @@ untrusted_stk_set(void *sp)
 	asm ("msr	rcsp_el0, %0" :: "C" (sp));
 }
 
+static inline void *
+untrusted_stk_get(void)
```

Comment:
stk_table_get, stk_table_set, untrusted_stk_set and then untrusted_stk_get is an unusual order, though I guess the benchmark ABI commit makes it less weird because only one becomes trusted